src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -396,66 +393,33 @@ def perform_post_processing(
             obs_dir = build_output_dir(final_output_dir, subdir=observer)
             seed_ext = f"_{seed}" if seed != "" else ""
             write_shard_metadata(observer, obs_data, obs_dir, seed_ext)
-            write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.{extension}")
+            write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.parquet")
 
 
 def load_data(raw_results_dir: Path, seed: str) -> Dict[str, pd.DataFrame]:
-    # Loads in all observer outputs and stores them in a dict with observer name as keys.
+    """Loads in all observer outputs and stores them in a dict with observer name as keys."""
     observers_results = {}
     for observer in FINAL_OBSERVERS:
-        obs_dir = raw_results_dir / observer
-        if seed == "":
-            logger.info(f"Loading {obs_dir.name} data ")
-            obs_files = sorted(
-                list(chain(*[obs_dir.glob(f"*.{ext}") for ext in SUPPORTED_EXTENSIONS]))
-            )
-            obs_data = []
-            for file in obs_files:
-                df = read_datafile(file)
-                df["random_seed"] = file.name.split(".")[0].split("_")[-1]
-                df = formatter.format_columns(df)
-                obs_data.append(df)
-            obs_data = pd.concat(obs_data)
-        else:
-            obs_files = list(
-                chain(*[obs_dir.glob(f"*_{seed}.{ext}") for ext in SUPPORTED_EXTENSIONS])
-            )
-            if len(obs_files) > 1:
-                raise FileExistsError(
-                    f"Too many files found with the given seed {seed} for observer {observer}"
-                    f" - {obs_files}."
-                )
-            elif len(obs_files) == 0:
-                raise FileNotFoundError(
-                    f"No file found with the seed {seed} for observer {observer}."
-                )
-            obs_file = obs_files[0]
-            logger.info(f"Loading data for {obs_file.name}")
-            obs_data = read_datafile(obs_file)
-            obs_data["random_seed"] = seed
-            obs_data = formatter.format_columns(obs_data)
-
-        observers_results[obs_dir.name] = obs_data
+        logger.info(f"Loading data for {observer}")
```

Comment:
This bit is the primary goal of this PR. We no longer have seed-specific raw results but rather a single file (per observer) with all seeds in it. So between that, dropping support for multiple file types (parquet AND hdf), and a small refactor to `read_datafile`, this all becomes simpler